src/mixin-bind-behaviour-test.js

```diff
@@ -100,9 +100,9 @@ if (browserSupports.customElements) {
 		customElements.define("setter-multiple-check", BasicBindingsElement);
 
 		const el = document.createElement('setter-multiple-check');
-		el.setAttribute('name', 'Kevin');
 
 		fixture.appendChild(el);
+		el.setAttribute('name', 'Kevin');
```

Comment:
After talking with @Mattchewone, I had to change this because I've broken how this works with `fromAttribute`.

Scenarios like this where an element is "upgraded" will not have their `fromAttribute` bindings triggered if the `stacheBindings.behaviors.initializeViewModel` is called in the `connect` hook:

```
<my-el name="Matt"></my-el>

<script type="module">
import { fromAttribute, StacheElement } from "//unpkg.com/can@pre/core.mjs";

class MyElement extends StacheElement {
    static view = `
        <p>{{this.name}}</p>
    `;
    static props = {
        name: { type: String, bind: fromAttribute }
    };
}
customElements.define("my-el", MyElement);
</script>
```